src/platform/plugins/shared/saved_objects_management/server/lib/find_relationships.ts

```diff
@@ -86,5 +86,7 @@ function extractCommonProperties(savedObject: SavedObjectWithMetadata) {
     id: savedObject.id,
     type: savedObject.type,
     meta: savedObject.meta,
+    ...(savedObject.managed !== undefined && { managed: savedObject.managed }),
```

Comment:
```suggestion
    managed: Boolean(savedObject.managed),
```
Nit: would it make sense to make `SavedObjectRelation.managed` required and just coerce it to a boolean here?

Reply:
@davismcphee thanks for your feedback!